tabu-app/src/App.vue

```diff
@@ -0,0 +1,91 @@
+<script setup lang="ts">
+import { RouterLink, RouterView } from 'vue-router'
+import HelloWorld from './components/HelloWorld.vue'
+</script>
+
+<template>
+  <header>
+    <img
+      alt="Vue logo"
+      class="logo"
+      src="@/assets/logo.svg"
+      width="125"
+      height="125"
+    />
+
+    <div class="wrapper">
+      <HelloWorld msg="You did it!" />
+
+      <nav>
+        <RouterLink to="/">Home</RouterLink>
+        <RouterLink to="/about">About</RouterLink>
+      </nav>
+    </div>
+  </header>
+
+  <RouterView />
+</template>
+
+<style scoped>
```

Comment:
Umjesto CSSa bilo bi dobro dodati SCSS. Ali to možda bolje prilikom prvog dodavanja 'stvarnih' stilova.

Reply:
I was thinking about something like this: https://tailwindcss.com/docs/guides/vite, have you tried this approach?

On previous projects I used SCSS, but most recently styled components in React, it seemed quite capable and easy to scale, however it might be a bit more complex than needed  https://styled-components.com/

Reply:
I wouldn't say I like remembering many classes to add some style, so I don't like Tailwind and similar frameworks. 

Also, why introduce styles via 'harder' JS? I believe that the application will be 'lighter' with as little JS as possible, and that includes more (S)CSS and less JS. 

But if Tailwind and styled-components will be accepted as a basis for the project, I will adapt 🙂

Reply:
For the start of project it would be best to start small and see what works, so yeah, lets use SCSS, possibly there is also some neat way to handle vendor prefixes? If that does not work, we can consider other approaches

Reply:
Ok, I agree 🙂 

Yes, there is https://github.com/postcss/autoprefixer that we can add to the project to take care of prefixes.